liblumen_core/src/sys.rs

```diff
@@ -20,29 +20,35 @@ pub use self::arch::alloc;
 pub use self::arch::mmap;
 
 pub mod sysconf {
+    use cfg_if::cfg_if;
     use lazy_static::lazy_static;
 
     use super::arch::sysconf;
 
-    /// The minimum alignment guaranteed by the target
-    #[cfg(all(any(
-        target_arch = "x86",
-        target_arch = "arm",
-        target_arch = "mips",
-        target_arch = "powerpc",
-        target_arch = "powerpc64",
-        target_arch = "asmjs",
-        target_arch = "wasm32"
-    )))]
-    pub const MIN_ALIGN: usize = 8;
-    #[cfg(all(any(
-        target_arch = "x86_64",
-        target_arch = "aarch64",
-        target_arch = "mips64",
-        target_arch = "s390x",
-        target_arch = "sparc64"
-    )))]
-    pub const MIN_ALIGN: usize = 16;
+    // The minimum alignment guaranteed by the target
+    cfg_if! {
+        if #[cfg(all(any(
+                target_arch = "x86",
+                target_arch = "arm",
+                target_arch = "mips",
+                target_arch = "powerpc",
+                target_arch = "powerpc64",
+                target_arch = "asmjs",
+                target_arch = "wasm32"
```

Comment:
This will ultimately be used to drive the minimum alignment for all allocations, so this list of targets either provides 8-byte minimum alignment via the global allocator, or we will request at least `MIN_ALIGN` alignment. I haven't done that yet though